mantidproject/mantid
# Indirect - ElasticWindowMultiple - Read temperature sensor names from instrument parameter files

Currently, the ElasticWindowMultiple algorithm contains constant values for the expected names of the sample changer temperature sensors. These values have recently been updated, resulting in ElasticWindowMultiple being unusable in its existing state.

The (correct) temperature sensor names have been moved to the instrument parameter files within this PR and ElasticWindowMultiple has been modified to read these names from the appropriate IPF file.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > Elwin.
2. Provide the supplied file in the 'Input File' field.
3. Ensure the 'Normalise by Temp.' check-box is checked.
4. Enter ```position``` into the SE Log Name field. 
5. Click 'Run'.
6. Ensure a 'Position not found in workspace.' warning is not produced.

**Test File:**
[iris77551_graphite002_red.zip](https://github.com/mantidproject/mantid/files/1781677/iris77551_graphite002_red.zip)



Fixes #21771 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
